src/core/p5.Renderer.js

```diff
@@ -208,18 +212,26 @@ p5.Renderer.prototype.textAlign = function(h, v) {
   }
 };
 
+p5.Renderer.prototype.textHyphens = function(bool) {
+  this._setProperty('_textHyphens', bool);
+  return this._textHyphens;
+};
+
+p5.Renderer.prototype.textWrap = function(wrapStyle) {
+  this._setProperty('_textWrap', wrapStyle);
+  return this._textWrap;
+};
+
 p5.Renderer.prototype.text = function(str, x, y, maxWidth, maxHeight) {
   const p = this._pInst;
-  let cars;
-  let n;
-  let ii;
-  let jj;
+  const textWrapStyle = this._textWrap;
+  const hyphenation = this._textHyphens;
+  let lines;
   let line;
   let testLine;
-  let currentLineLength;
   let testWidth;
   let words;
-  let totalHeight;
+  let chars;
```

Comment:
i like this rename! :)

Reply:
this was actually not a rename, just a declaration of a different variable in the same line where I removed one that was unused. totalHeight was only used in the first of the line-splitting for loops (removed from lines 240-261). the other line-splitting for loop that happens later uses y, which actually moves the "cursor" down to the next line by adding the height of the previous line. 

it was very strange to me that this whole testing-and-splitting lines operation was happening twice in this file (lines 240-261 and again in 297-308) with differently named variables - esp. because the first one with currentLineLength and totalHeight seemed have no bearing on what was rendered, not being added to the actual lines or y-axis calculations 🤔

Reply:
> this was actually not a rename,

ah sorry i meant renaming `chars` to `lines` for use on the now line 250! i think i selected the wrong lines for my comment 🤦🏻‍♀️

> it was very strange to me that this whole testing-and-splitting lines operation was happening twice in this file 

i get where you're coming from but tbh its not _that_ surprising to me. a lot of people have touched this logic over the last 6 years and its probably been a while since anyone looked at this function holistically.